decidim-core/lib/decidim/events/simple_event.rb

```diff
@@ -71,9 +71,16 @@ def default_i18n_options
           resource_path: resource_path,
           resource_title: resource_title,
           resource_url: resource_url,
+          participatory_space_title: participatory_space_title,
           scope: i18n_scope
         }
       end
+
+      def participatory_space_title
+        return unless participatory_space
+
+        participatory_space.title[I18n.locale.to_s]
```

Comment:
I think it'd be nice if we used this here:

https://github.com/decidim/decidim/blob/master/decidim-core/app/helpers/decidim/translations_helper.rb#L13

It's a little cleverer as it tries some fallbacks, what do you think?